--- gnome-scan-0.7.1/src/flegita-gimp.c~	2008-02-27 22:02:30.000000000 +0100
+++ gnome-scan-0.7.1/src/flegita-gimp.c	2009-01-15 06:21:13.000000000 +0100
@@ -134,6 +134,7 @@
   gimp_ui_init (PACKAGE, FALSE);
   gnome_scan_init(NULL, NULL);
   textdomain (GETTEXT_PACKAGE);
+  bind_textdomain_codeset (GETTEXT_PACKAGE,"UTF-8");
   gdouble xres, yres, res;
 
   /* if we are scanning, create a new image */
